app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java

```diff
@@ -326,6 +328,22 @@ public void onClick(DialogInterface dialog, int which) {
         }
     }
 
+    private void checkInstagram() {
+            PackageManager packageManager = (ActivitySwitchHelper.context).getPackageManager();
+            if(Utils.isAppInstalled(PACKAGE_INSTAGRAM,packageManager)){
+                realm.beginTransaction();
+                account = realm.createObject(AccountDatabase.class, INSTAGRAM.toString());
+                account.setAccountname(INSTAGRAM);
+                account.setUsername(INSTAGRAM.toString());
+                realm.commitTransaction();
```

Comment:
@mohitmanuja Yes, I know. Currently, there is an option for Instagram in the Accounts Activity, which I found it useless as well. So I added this functionality.